### DISCUSSION TITLE
`vscode.git` API on NPM

### DISCUSSION THREAD

**Discussion by Malix-Labs** [Security-Experts-Community/vscode-xp#169] [General]
Hey guys!

I recently discovered you were also using the `vscode.git` API;
Meaning you face the same problem as me which is to deal with its terrible delivery (see https://github.com/microsoft/vscode/issues/198592).

I feature-requested a better delivery (publish on NPM, basically) here: https://github.com/microsoft/vscode/issues/198592.

It technically needs 20 👍 to be accepted into the backlog, so I'm sharing it with you if you find interesting

Thanks and have a very great day!

**Comment by DmitryOffsec** [comment]
Hello! Great idea, I'll give it a like
